ansible/roles/datadog/templates/riakcs.yaml.j2

```diff
@@ -0,0 +1,9 @@
+init_config:
+
+# key values are set by ../../roles/riakcs/sync_keys/main.yml
+instances:
+  - host: {{ inventory_hostname if inventory_hostname|ipaddr else lookup('dig', inventory_hostname, wantlist=True)[0] }}
+    port: {{ riakcs_port }}
+    is_secure: False
+    access_id: access-key
+    access_secret: access-secret
```

Comment:
This config appears to be correct, but the datadog riakcs integration is still reporting an error. See https://github.com/DataDog/dd-agent/issues/2340 for more details.